utam-compiler/src/main/java/utam/compiler/translator/DefaultSourceConfiguration.java

```diff
@@ -85,11 +85,13 @@ void preProcess(String packageName, Path filePath, String fileMaskRegex) {
   }
 
   String getPageObjectURI(String packageName, Path filePath, String fileMaskRegex) {
-    final Pattern relativePattern = Pattern.compile(fileMaskRegex);
+    final Pattern relativePattern = Pattern.compile(
+        fileMaskRegex.replace("/", Matcher.quoteReplacement(File.separator)));
```

Comment:
I would probably add static method for ```replace("/", Matcher.quoteReplacement(File.separator))``` since it's reused in multiple places

Reply:
Fixed in subsequent commit.